src/microbe_stage/editor/MicrobePartSelection.tscn

```diff
@@ -96,9 +96,24 @@ mouse_filter = 2
 size_flags_vertical = 0
 expand = true
 stretch_mode = 6
-__meta__ = {
-"_edit_use_anchors_": false
-}
+
+[node name="RecentlyUnlocked" type="MarginContainer" parent="VBoxContainer/Button"]
+anchor_left = 1.0
+anchor_right = 1.0
+margin_left = -30.0
+margin_bottom = 14.0
+grow_horizontal = 0
+
+[node name="ColorRect" type="ColorRect" parent="VBoxContainer/Button/RecentlyUnlocked"]
+margin_right = 30.0
+margin_bottom = 14.0
+color = Color( 0.741176, 0.356863, 0.917647, 1 )
+
+[node name="Label" type="Label" parent="VBoxContainer/Button/RecentlyUnlocked"]
+margin_right = 30.0
+margin_bottom = 14.0
+text = "NEW"
```

Comment:
I think the translation key should be more specific here like "UNLOCKED_NEW_ITEM". Also the uppercase specification here is something that isn't used elsewhere in the game, rather all the text uppercasing is handled with translations.